manageRGsAndDeployments.go

```diff
@@ -183,10 +183,56 @@ func (m *MinPermFinder) DeployARMTemplate(deploymentName string) error {
 	log.Debugln()
 	// create JSON body with template and parameters
 
-	url := fmt.Sprintf("https://management.azure.com/subscriptions/%s/resourcegroups/%s/providers/Microsoft.Resources/deployments/%s?api-version=2020-10-01", m.SubscriptionID, m.ResourceGroupName, deploymentName)
```

Comment:
Can we have the default option as "what if" , but also retain the full deployment option? The reason is that the idea is to support terraform files (other than ARM and bicep) in the future, and I am not sure the what if option will work with the "what if" analysis